astronomer/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -43,6 +43,30 @@ def execute(self, context: Context) -> None:  # type: ignore[override]
         if response.get("status") == "error":
             self.execute_complete(context, event=response)
         context["ti"].xcom_push(key="return_value", value=query_ids)
+
+        still_running = False
+        completed_query_ids = []
+        for qid in query_ids:
+            resp = redshift_data_hook.conn.describe_statement(Id=qid)
+            status = resp["Status"]
+            if status == "FAILED":
+                err_msg = f"Error: {resp['QueryString']} query Failed due to {resp['Error']}"
```

Comment:
Can we use format specifiers instead of f-strings

Reply:
I would prefer f-strings in general unless it is used for logging in self.log.....